apps/webapp/app/dashboard/types.ts

```diff
@@ -11,3 +11,11 @@ export type DashboardTabMap = Record<
     content: EduPanel;
   }
 >;
+
+export type DashboardMetadataMap = Record<
+  DashboardTab,
+  {
+    title: string;
+    description: EduPanel.ASSETS | EduPanel.TRANSACTIONS_LIST | EduPanel.TEMP_FILLER;
```

Comment:
This will end up rendering: `0` | `1` | `5`

Also, test in your browser to see the html:
```
<meta name="description" content="0">
```

Reply:
You should be using: `eduPanelContent`

Reply:
I will be more careful in the future